### ISSUE TITLE
Choice group buttons have discernible text

### ISSUE THREAD

**Issue by ipip2005** [microsoft/fluentui#17278]
### Environment Information

OS: Win 10 (OS Build 21313.1000)
Browser : Edge Dev (Version 90.0.796.0 (Official build) dev (64-bit))
Tool : Accessibility Insight for Web

### Describe the issue:

Go to https://developer.microsoft.com/en-us/fluentui#/controls/web/choicegroup
Inspect the option.

#### Actual behavior:

Actual: input is associated with a label element which has `for` attribute pointing to the input.
Expected: 

#### Expected behavior:

Fix ONE of the following:
Element does not have inner text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element's default semantics were not overridden with role="none" or role="presentation"
Element has no title attribute

### Documentation describing expected behavior

MAS Reference Link:
[MAS4.1.2-Name, Role and Value](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7bB661D774-60D0-4277-A121-73F986EBEE76%7d&file=MAS%204.1.2%20%E2%80%93%20Name%2c%20Role%2c%20Value.docx&wdLOR=cE56E09CE-4478-4D20-B785-78D1B90569C1&action=default&mobileredirect=true&cid=6a906cac-bcff-4869-ae0c-aecbb5cee6ec)

**Comment by gouttierre** [comment]
@ipip2005 - Thanks for reach out. 

@khmakoto - Would you be able to confirm if the desired expected behaviors were missed during regression, or if this behavior is an issue?

**Comment by khmakoto** [comment]
Hi @ipip2005 this doesn't repro for me on the website as you can see in the screenshot below. Could you take a look again?

![image](https://user-images.githubusercontent.com/7798177/111848705-f75f8080-88c8-11eb-8bcc-b83cd902b8a2.png)
